tests/network/bond/test_bond_modes.py

```diff
@@ -201,7 +202,10 @@ def test_bond_created(workers_utility_pods, matrix_bond_modes_bond):
 
 @pytest.mark.polarion("CNV-4383")
 def test_vm_started(bond_modes_vm):
-    running_vm(vm=bond_modes_vm, check_ssh_connectivity=False, wait_for_interfaces=False)
+    bond_modes_vm.start(wait=True)
+    bond_modes_vm.vmi.wait_for_condition(
+        condition=Resource.Condition.Type.AGENT_CONNECTED, status=Resource.Condition.Status.TRUE
+    )
```

Comment:
maybe consider adding new method to VMI in wrapper `wait_for_agent_connected`?

Reply:
Makes sense considering we need to do this all over the network codebase.

Can I follow up on this in a separate PR? There is another improvement requested already.
I will do that before continuing with the rest of similar changes.

Reply:
Ack for follow-up PR